src/bin/index.ts

```diff
@@ -77,29 +77,33 @@ import { getConfigs } from '../parsers/options.js';
   if (hasArg('list-files')) {
     const { listFiles } = require('../modules/helpers/list-files.js');
 
-    let total = 0;
+    const files: string[] = [];
 
     Write.hr();
 
     for (const dir of dirs) {
-      const files: string[] = await listFiles(dir, {
-        filter:
-          typeof filter === 'string'
-            ? new RegExp(escapeRegExp(filter))
-            : filter,
-        exclude:
-          typeof exclude === 'string'
-            ? new RegExp(escapeRegExp(exclude))
-            : exclude,
-      });
-
-      total += files.length;
-
-      Write.log(files.map((file) => `${format('-').dim()} ${file}`).join('\n'));
+      files.push(
+        ...(await listFiles(dir, {
```

Comment:
Hm.. What if `listFiles()` would take `string | string[]`?

Reply:
There would be three motivations for using array: 

1. Avoids extra conditions (e.g., `if ... isArray` or `if ... string`)
2. Reduces coverage possibilities
3. The plural name indicates that the return should be a list

This combination helps both to keep use easy and consistent for the end user, and also to maintain it 🙋🏻‍♂️

Reply:
I've just realized that I took that in the wrong way (I considered the return rather than the parameters received) 😅

It would be interesting 🙋🏻‍♂️